alerts/kueue/kueue-pod-down.md

```diff
@@ -0,0 +1,42 @@
+# Kueue Pod Down
+
+## Severity: Critical
+
+## Impact
+
+Any workloads running on the cluster will not be able to use the Kueue component.
+
+## Summary
+
+This alert is triggered when the `kube_pod_status_ready` query shows that the Kueue controller pod is not ready.
+
+## Steps
+
+1. Check to see if the `kueue-controller` pod is running in the `redhat-ods-applications` namespace:
+
+```bash
+oc -n redhat-ods-applications get pods -l app.kubernetes.io/name=kueue
+```
+
+2. If the pod is not running, look at the pod's logs/events to see what may be causing the issues. Please make sure to grab the logs/events so they can be shared with the engineering team later:
+
+```bash
+# Check pod logs 
+oc -n redhat-ods-applications logs -l app.kubernetes.io/name=kueue --prefix=true
```

Comment:
This command shows just last 10 lines.
Do we rather want to retrieve whole log file? For that user can use:
```
oc -n redhat-ods-applications logs -l app.kubernetes.io/name=kueue --prefix=true --tail=-1
```